client/src/views/clan/ClanView.tsx

```diff
@@ -733,12 +733,12 @@ const ClanView = () => {
                       </TableHead>
                       <TableBody>
                         {clan.bossDamages.map((damages, index) => (
-                          <TableRow key={damages.brute.id}>
+                          <TableRow key={damages.brute?.id}>
```

Comment:
React keys should never be the same, it will fail here if two brutes were deleted in the same clan.
Instead, you can simply filter the boss damages from deleted brute, both here and in the backend, to be sure